9.2/root/usr/bin/run-postgresql

```diff
@@ -16,5 +16,10 @@ fi
 
 set_passwords
 
+if [ -f ${CONTAINER_SCRIPTS_PATH}/post-init.sh ]; then
+  log_info 'Sourcing post-init.sh ...'
+  source ${CONTAINER_SCRIPTS_PATH}/post-init.sh
```

Comment:
I recently came across that code in MySQL, it's not documented anywhere, so perhaps it's our chance to review how it is implemented and document it :)